tests/ses/deepsea_cluster_deploy.pm

```diff
@@ -38,22 +37,41 @@ sub run {
         # salt does not return 1 if any node will fail ping test
         assert_script_run 'for i in {1..7}; do echo "try $i" && if [[ $(salt \'*\' test.ping |& tee ping.log) = *"Not connected"* ]];
  then cat ping.log && false; else salt \'*\' test.ping && break; fi; done';
-        assert_script_run 'salt \'*\' cmd.run \'lsblk\'';
-        assert_script_run 'deepsea stage run ceph.stage.0', 1200;
-        assert_script_run 'deepsea stage run ceph.stage.1', 1200;
+        assert_script_run "salt \'*\' cmd.run \'lsblk\' |& tee /dev/$serialdev";
```

Comment:
Just for info, you are checking return value of `&| tee` (last piped command)
If you really wanted to check value (probably not this case anyway), I use this:
```
assert_script_run ".....commands | tee....; ( exit \${PIPESTATUS[0]} )");
assert_script_run ".....commands | tee.... | grep 'SALT_OK'"

Reply:
I'm just printing disks